src/ensembl/production/dbcopy/models.py

```diff
@@ -180,6 +185,25 @@ def _clean_db_set_for_filters(self, from_host, field):
         except ValueError as e:
             raise ValidationError({field: str(e)})
 
+    @staticmethod
```

Comment:
To be inline with Django Framework, this method shouldn't be located within RequestJob Model class, but in a dedicated `Manager` one. 
```
class RequestJobManager(models.Manager):
        def equivalent_running_jobs(self, **kwargs):
           pass
```

```
class RequestJob(models.Model):
 ...
   objects = RequestJobManager()
```

Then you'll call it with

Reply:
Good idea, thanks for the tip! See e529355